samples/browser/src/main/java/org/mozilla/samples/browser/CustomTabActivity.kt

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser
+
+
+class CustomTabActivity : BrowserActivity() { }
```

Comment:
\[ktlint\] WARNING: \[no\-empty\-class\-body\] Unnecessary block \("\{\}"\) in column 45